test/integration/.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+
+  // jest implicit/reserved words
+  globals: {
+    describe: "readonly",
+    it: "readonly",
+    beforeAll: "readonly",
+    beforeEach: "readonly",
+    afterAll: "readonly",
+    afterEach: "readonly",
+  },
+
+  // some style exceptions for stuff that's useful in tests
+  rules: {
+    "no-console": 0,          // extra logging, obviously
```

Comment:
I do have a mild preference for just removing the messages and just adding logs back in if/when they're needed for debugging, but if you're keen on leaving these in then IMO this should remain as-is in favour of using `debug` for the test-suite specific debug-level messages. 

Using `debug` with a suite-specific prefix would hide this by default e.g. consider what happens to the signal vs noise of these messages if other tests running in parallel also want to log something like `'starting beforeEach'`. `debug` is already a dependency.